packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -44,21 +46,48 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
   assert('PoolShares' in terms.brands, 'no PoolShares brand');
 
   const statusManager = prepareStatusManager(zone);
-  const feed = prepareTransactionFeed(zone);
   const makeSettler = prepareSettler(zone, { statusManager });
   const { chainHub, vowTools } = tools;
   const makeAdvancer = prepareAdvancer(zone, {
     chainHub,
-    feed,
     log: trace,
     statusManager,
     vowTools,
   });
-  assertAllDefined({ feed, makeAdvancer, makeSettler, statusManager });
+  const makeFeedKit = prepareTransactionFeedKit(zone);
+  assertAllDefined({ makeFeedKit, makeAdvancer, makeSettler, statusManager });
+  const feedKit = makeFeedKit();
+  const advancer = makeAdvancer(
+    // @ts-expect-error FIXME
+    {},
+  );
+  // Connect evidence stream to advancer
+  void observeIteration(subscribeEach(feedKit.public.getEvidenceStream()), {
+    updateState(evidence) {
+      advancer.handleTransactionEvent(evidence);
+    },
+  });
 
   const creatorFacet = zone.exo('Fast USDC Creator', undefined, {});
 
-  return harden({ creatorFacet });
+  const publicFacet = zone.exo('Fast USDC Public', undefined, {
+    // XXX to be removed before production
+    /**
+     * @param {CctpTxEvidence} evidence
+     */
+    makeTestPushInvitation(evidence) {
+      return zcf.makeInvitation(
```

Comment:
Maybe we can try without an actual invitation, just a method call?

Reply:
That's how Patrick's PR already works. The point of this PR is so the CLI can push evidence. That can only be done through an offer through the smart wallet

Reply:
I was thinking more about the workaround we discussed where the smart wallet would invoke the invitation maker, but the invitation maker itself would just throw without returning an invitation. To avoid the perf overhead of zoe. We'd still want this exo for managing oracle members and aggregating submissions

Reply:
>I was thinking more about the workaround we discussed where the smart wallet would invoke the invitation maker,

I see. I'll give that a try.